tests/varres/tprevent_forloopvar_mutations.nim

```diff
@@ -1,7 +1,7 @@
 discard """
   errormsg: "type mismatch: got <int>"
   line: 17
```

Comment:
do we need those 2 lines, given that we already capture
```
nimout: '''tprevent_forloopvar_mutations.nim(17, 7) Error: type mismatch: got <int>
```

I'd prefer simplicity here and in other tests; specifying separately line, col, errormsg is more work and nimout already captures what we want to test.

Ditto in other examples above, eg, replacing:
```
  errormsg: "undeclared identifier: 'undeclared'"
  line: 10
  column: 6
```
by:
```
  nimout: '''tcolumn.nim .nim(10, 6) undeclared identifier: 'undeclared''''
```

and similar examples

Reply:
ok

Reply:
then I need to change line number which is a huge work can be left in the future PR.

Reply:
yes can be deferred


> then I need to change line number

no, you could just add empty newlines